### PR TITLE
Nuget Title & Description for Abstractions

### LINKED ISSUE #6404: Description for NuGet packages
All packages just have "Package Description" now. However, we don't want to write descriptions for all individual packages (unless someone is masochistic enough) so a better thing would be to generate it somehow.

- For projects with a Manifest include the descriptions from there.
- For other projects use a common template.

In both cases, the text could include something like:

> Orchard Core is an open-source, modular, multi-tenant application framework and CMS for ASP.NET Core. For detailed information about Orchard Core, and for documentation on how to use it and integrate it into your app see https://www.orchardcore.net/.

### PR DESCRIPTION
**Pull request by ns8482e**
Abstractions

## Orchard Core Framework

- [x] OrchardCore
- [x] OrchardCore.Abstractions
- [x] OrchardCore.Apis.GraphQL.Abstractions
- [x] OrchardCore.Apis.GraphQL.Client
- [x] OrchardCore.DisplayManagement
- [x] OrchardCore.DisplayManagement.Abstractions
- [x] OrchardCore.DisplayManagement.Liquid
- [x] OrchardCore.DynamicCache.Abstractions
- [x] OrchardCore.Email.Abstractions
- [x] OrchardCore.Feeds.Abstractions
- [x] OrchardCore.Feeds.Core
- [x] OrchardCore.FileStorage.Abstractions
- [x] OrchardCore.FileStorage.AzureBlob
- [x] OrchardCore.FileStorage.FileSystem
- [x] OrchardCore.Liquid.Abstractions
- [x] OrchardCore.Localization.Abstractions
- [x] OrchardCore.Localization.Core
- [x] OrchardCore.Navigation.Core
- [x] OrchardCore.Recipes.Abstractions
- [x] OrchardCore.Recipes.Core
- [x] OrchardCore.Redis.Abstractions
- [x] OrchardCore.XmlRpc.Abstractions
- [x] OrchardCore.ResourceManagement.Abstractions
- [x] OrchardCore.Scripting.JavaScript
- [x] OrchardCore.Shells.Azure
- [x] OrchardCore.Logging.NLog
- [x] OrchardCore.Logging.Serilog
- [x] OrchardCore.Mvc.Core

### Module
- [x] OrchardCore.Apis.GraphQL
- [x] OrchardCore.BackgroundTasks
- [x] OrchardCore.DataProtection.Azure
- [x] OrchardCore.Diagnostics
- [x] OrchardCore DynamicCache
- [x] OrchardCore.Email
- [x] OrchardCore.Features
- [x] OrchardCore.Feeds
- [x] OrchardCore.HealthChecks
- [x] OrchardCore.Https
- [x] OrchardCore.Localization
- [x] OrchardCore.MiniProfiler
- [x] OrchardCore.Navigation
- [x] OrchardCore.Recipes
- [x] OrchardCore.Redis
- [x] OrchardCore.Resources
- [x] OrchardCore.ResourceManagement
- [x] OrchardCore.ResponseCompression
- [x] OrchardCore.ReverseProxy
- [x] OrchardCore.Scripting
- [x] OrchardCore.XmlRpc

## Orchard Core CMS

- [x] OrchardCore.Admin.Abstractions
- [x] OrchardCore.AdminMenu.Abstractions
- [x] OrchardCore.ContentLocalization.Abstractions
- [x] OrchardCore.ContentManagement
- [x] OrchardCore.ContentManagement.Abstractions
- [x] OrchardCore.ContentManagement.Display
- [x] OrchardCore.ContentManagement.GraphQL
- [x] OrchardCore.ContentPreview.Abstractions
- [x] OrchardCore.Contents.Core
- [x] OrchardCore.Contents.TagHelpers
- [x] OrchardCore.ContentTypes.Abstractions
- [x] OrchardCore.Data
- [x] OrchardCore.Data.Abstractions
- [x] OrchardCore.Deployment.Abstractions
- [x] OrchardCore.Deployment.Core
- [x] OrchardCore.Indexing.Abstractions
- [x] OrchardCore.Lucene.Abstractions
- [x] OrchardCore.Lucene.Core
- [x] OrchardCore.Markdown.Abstractions
- [x] OrchardCore.Media.Abstractions
- [x] OrchardCore.Media.Core
- [x] OrchardCore.MetaWeblog.Abstractions
- [x] OrchardCore.OpenId.Core
- [x] OrchardCore.Queries.Abstractions
- [x] OrchardCore.ReCaptcha.Core
- [x] OrchardCore.Search.Abstractions
- [x] OrchardCore.Setup.Abstractions
- [x] OrchardCore.Setup.Core
- [x] OrchardCore.ShortCodes.Abstractions
- [x] OrchardCore.Settings.Core
- [x] OrchardCore.Sitemaps.Abstractions
- [x] OrchardCore.Users.Abstractions
- [x] OrchardCore.Users.Core
- [x] OrchardCore.Workflows.Abstractions
- [x] OrchardCore.Infrastructure
- [x] OrchardCore.Infrastructure.Abstractions

### Module

- [x] OrchardCore.Admin
- [x] OrchardCore.AdminMenu
- [x] OrchardCore.Alias
- [x] OrchardCore.Autoroute
- [x] OrchardCore.ContentFields
- [x] OrchardCore.ContentLocalization
- [x] OrchardCore.ContentPreview
- [x] OrchardCore.Contents
- [x] OrchardCore.ContentTypes
- [x] OrchardCore.Deployment
- [x] OrchardCore.Deployment.Remote
- [x] OrchardCore.Facebook
- [x] OrchardCore.Flows
- [x] OrchardCore.Forms
- [x] OrchardCore.Github
- [x] OrchardCore.Google
- [x] OrchardCore.HomeRoute
- [x] OrchardCore.Html
- [x] OrchardCore.Indexing
- [x] OrchardCore.Layers
- [x] OrchardCore.Lists
- [x] OrchardCore.Lucene
- [x] OrchardCore.Markdown
- [x] OrchardCore.Media
- [x] OrchardCore.Media.Azure
- [x] OrchardCore.Microsoft.Authentication
- [x] OrchardCore.OpenId
- [x] OrchardCore.Placements
- [x] OrchardCore.PublishLater
- [x] OrchardCore.Queries
- [x] OrchardCore.ReCaptcha
- [x] OrchardCore.Roles
- [x] OrchardCore.Setup
- [x] OrchardCore.ShortCodes
- [x] OrchardCore.Settings
- [x] OrchardCore.CustomSettings
- [x] OrchardCore.Sitemaps
- [x] OrchardCore.Taxonomies
- [x] OrchardCore.Templates
- [x] OrchardCore.Tenants
- [x] OrchardCore.Themes
- [x] OrchardCore.Title
- [x] OrchardCore.Twitter
- [x] OrchardCore.Users
- [x] OrchardCore.Widgets
- [x] OrchardCore.Workflows

### Targets

- [x] OrchardCore.Application.Mvc.Targets
- [x] OrchardCore.Application.Targets
- [x] OrchardCore.Application.Cms.Core.Targets
- [x] OrchardCore.Application.Cms.Targets
- [x] OrchardCore.Module.Targets
- [x] OrchardCore.Theme.Targets

### Code Templates
- [x] OrchardCore.ProjectTemplates

### Samples 
- [x] OrchardCore.Demo
- [x] OrchardCore.Mvc.HelloWorld

Fixes #6404

### CODE REVIEW

**Comment by agriffard** [comment]
The remaining todos are mostly about core and abstractions modules.
Do you intend to add a title and a description for all of them?

We will merge your other PR #7237 soon and this one is kind of linked to it so please tell us when you thing it can be ready.

**Comment by ns8482e** [comment]
> Do you intend to add a title and a description for all of them?

Yes

**Comment by sebastienros** [comment]
About tags, I don't see the point of adding part of the title as a tag, if there is no common tag between each of them. It won't make it more searchable. Commons tags liks "cms, theme, module, orchardcore" is fine. But "Remote" doesn't add any value. It's just tokenizing the title. And it makes me sad because it's you own time that I waste :/

**Comment by ns8482e** [comment]
@sebastienros there is no `Remote` tag, Following are the the common tags are used 

OrchardCoreFramework 
OrchardCoreCMS 
Localization
ContentManagement
DisplayManagement
OpenId
Infrastructure
Abstractions
GraphQL 
GraphQL Client 
Data 
Liquid
Cache
RSS
Azure

File: src/OrchardCore.Modules/OrchardCore.ContentPreview/OrchardCore.ContentPreview.csproj [hunk 1]
```diff
@@ -3,6 +3,12 @@
   <PropertyGroup>
     <TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <!-- NuGet properties-->
+    <Title>OrchardCore ContentPreview</Title>
+    <Description>$(OCCMSDescription)
+
+    Provides live content edition and content preview.</Description>
```
Comment (sfmskywalker) [inline src/OrchardCore.Modules/OrchardCore.ContentPreview/OrchardCore.ContentPreview.csproj:10]: "edition" -> "editing"

File: src/OrchardCore.Modules/OrchardCore.Deployment.Remote/OrchardCore.Deployment.Remote.csproj [hunk 2]
```diff
@@ -3,6 +3,12 @@
   <PropertyGroup>
     <TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+      <!-- NuGet properties-->
+    <Title>OrchardCore Deployment Remote</Title>
```
Comment (sfmskywalker) [inline src/OrchardCore.Modules/OrchardCore.Deployment.Remote/OrchardCore.Deployment.Remote.csproj:7]: "Depolyment Remote" sounds a bit funny. Should it be "OrchardCore Remote Deployment" instead?

File: src/OrchardCore.Modules/OrchardCore.ContentPreview/OrchardCore.ContentPreview.csproj (hunk 1 above)
Comment (ns8482e) [reply src/OrchardCore.Modules/OrchardCore.ContentPreview/OrchardCore.ContentPreview.csproj]: done

File: src/OrchardCore.Modules/OrchardCore.Deployment.Remote/OrchardCore.Deployment.Remote.csproj (hunk 2 above)
Comment (ns8482e) [reply src/OrchardCore.Modules/OrchardCore.Deployment.Remote/OrchardCore.Deployment.Remote.csproj]: done

**Comment by deanmarcussen** [comment]
Where are we at with this Niraj? (lot of work!) Have you updated to include all the projects, including the new ones now?

**Comment by ns8482e** [comment]
I guess there might be left out few new projects that recently added and are not listed in description of this PR

**Comment by ns8482e** [comment]
@deanmarcussen thank you for reviewing. I missed some of the modules that didn't have abstractions projects. Added them into todo list.

**Comment by Skrypt** [comment]
Is there anything missing in this PR?

**Comment by ns8482e** [comment]
@Skrypt @deanmarcussen I am done with the changes.

**Review by sebastienros — APPROVED** [review]

**Review by Skrypt — APPROVED** [review]

**Pull request merged**